js/pulse.js

```diff
@@ -50,10 +57,14 @@
              */
             this.initialize = function(adManagerController, playerId) {
                 amc = adManagerController; // the AMC is how the code interacts with the player
+                //Enable overlay close button right away for Pulse
+                amc.adManagerSettings["showNonLinearCloseButton"] = true;
```

Comment:
Changing this value will affect all ad managers. I'll update the sendURLToLoadAndPlayNonLinearAd function to include a parameter to force the AMC to publish the OO.EVENTS.SHOW_NONLINEAR_AD_CLOSE_BUTTON.